team-bayou/UNO-Shareboard
# Adds support for Versioning in entities now. Also adds update user fu…

To test:
- import postman collection update
- under user, try out the update user and use a get to test if the data changes
- also check in database to ensure version number in now being incremented
